WeatherAPI/WeatherAPI.csproj.user

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This should have been in .gitignore, but go ahead and add it again. 
You can copy the one from github too.